cfgov/unprocessed/css/organisms/email-popup.less

```diff
@@ -67,7 +67,7 @@
     }
 
     .respond-to-max( @bp-xs-max, {
-        background: none;
+        background: none !important;
```

Comment:
This is to override the inline background-image? Perhaps make it specifically `background-image: none !important;`

Reply:
@anselmbradford I'll defer to your frontend knowledge on this; would that cause any issues because it would leave [these other attributes set](https://github.com/cfpb/cfgov-refresh/pull/3731/files/255f54268d76d6386bda175e2a53170707861a2c#diff-dc1434e393a4b592cabc175ec1226462R83), or are those two sections mutually exclusive?

Reply:
Those _should_ be fine, because they'd only apply to an image set via `background-image`. It's probably fine as-is, but maybe add a code comment above it that says why the `!important` is needed, as important usage is discouraged by linters and such.

Reply:
Thanks @anselmbradford, addressed in f7ecede.